src/keria/app/agenting.py

```diff
@@ -108,6 +110,31 @@ def setup(name, bran, adminPort, bootPort, base='', httpPort=None, configFile=No
     return doers
 
 
+def createHttpServer(port, app, keypath, certpath, cafilepath):
```

Comment:
This method probably needs a test to get the patch coverage up

Reply:
I'll add one.